failFast

```diff
@@ -1 +1 @@
-true
+false
```

Comment:
Please keep in draft until this is reverted.

Reply:
```suggestion
true
```